### DISCUSSION TITLE
add a new deliverable GitHub profile

### DISCUSSION THREAD

**Discussion by sammou00** [HackYourFutureBelgium/workflows#6] [Ideas]
- [x] add `GitHub profile` deliverable

**Comment by sammou00** [comment]
@talmurshidi  What do you think?

**Reply by talmurshidi** [reply]
What is the definition of `GitHub profile`? 
what the learner should do?
Is it same as [GitHub setup](https://github.com/HackYourFutureBelgium/precourse/blob/main/materials/github-setup.md)?

**Reply by sammou00** [reply]
https://zzetao.github.io/awesome-github-profile/

**Reply by sammou00** [reply]
They should create a new repo named exactly like their GitHub user name.
then add info about themselves in the main `README.md`

**Reply by talmurshidi** [reply]
It is part of `precourse` module, not `workflow` module. Also, it was in technical assignment.

**Reply by sammou00** [reply]
Really? It should be part of the Workflows. Do you think it will be better to move it to Workflows deliverables?

**Reply by talmurshidi** [reply]
Workflows module is 2 weeks, maybe you can merge it with `Student Bio Repo` and keep it in `precourse` as well. Maybe make it optional

**Reply by sammou00** [reply]
Ok, then I will make it optional.
